README.md

```diff
@@ -1,5 +1,65 @@
 # Cloud Run Progressive Delivery Operator
 
+A gradual rollout operator for Cloud Run services.
+
+## How does it work?
+
+The operator provides an automated way to gradually roll out new versions of
+your code by increasing the traffic to your newly deployed revision. The
+operator periodically checks for new revisions in the services that opted-in for
+gradual rollouts. If a new revision with no traffic is found, the operator
+automatically assigns it some initial traffic. This new revision is labeled
+*candidate* while the previous revision serving traffic is labeled *stable*.
+
+After some time, the operator checks for metrics for the new revision. Using the
+set health criteria, it evaluates what to do next:
+
+1) If the metrics show a *healthy* revision, traffic to the *candidate* is
+increased.
+2) If the metrics show an *unhealthy* revision, traffic to the *candidate* is
+dropped and the *stable* revision gets 100% of the traffic.
+
+Once a healthy *candidate* handles 100% of the traffic, it becomes the new
+*stable* revision.
+
+## Usage
+
+`crun-release-operator -cli -project=myproject -interval=600`
```

Comment:
Just thinking about this, I realized that making `-run-interval=60s` (even as an example) might be an issue. This will mean that if the metrics show a healthy candidate, the candidate will become stable very quickly (~7 minutes) given our default config and the metrics delay from Cloud Monitoring. 
The major issue is that if we look for metrics in the last 30 minutes, but check for metrics every 1 minute, the weight of the metrics from the previous rollout process is pretty big (29 minutes). This doesn't allow balancing out if issues arrived because of request load.